lib/power-canvas.coffee

```diff
@@ -49,7 +49,7 @@ module.exports =
 
   getColorAtPosition: (left, top) ->
     offset = @editorElement.getBoundingClientRect()
-    el = atom.views.getView(@editor).shadowRoot.elementFromPoint(
+    el = (atom.views.getView(@editor).shadowRoot ? document).elementFromPoint(
```

Comment:
why document here? 
mm I don't know why are we getting the editor view `@editorElement` again if it is above :confused: